### PR TITLE
SOC-757 | Use slave for fetching notifications count

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-757

### CODE REVIEW

File: extensions/wikia/WallNotifications/WallNotifications.class.php [hunk 1]
```diff
@@ -39,10 +39,11 @@ public function getWikiNotifications( $userId, $wikiId, $readSlice = 5, $counton
 
 		// try fetching the list of notifications from memcache
 		$list = $memcSync->get();
+		$list = false;
```
Comment (macbre) [inline extensions/wikia/WallNotifications/WallNotifications.class.php]: Debugging code?

Comment (kvas-damian) [inline extensions/wikia/WallNotifications/WallNotifications.class.php]: nice catch! Thanks

**Comment by bkoval** [comment]
LGTM :+1:

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**